images/virtualization-artifact/pkg/controller/vm/internal/lifecycle.go

```diff
@@ -91,7 +91,12 @@ func (h *LifeCycleHandler) Handle(ctx context.Context, s state.VirtualMachineSta
 		return reconcile.Result{}, err
 	}
 
-	changed.Status.Phase = getPhase(kvvm)
+	phase := getPhase(kvvm)
+	if phase == "" {
+		phase = virtv2.MachinePending
+		log.Error(fmt.Sprintf("unexpected KVVM state: status %q, fallback VM phase to %q", kvvm.Status.PrintableStatus, phase))
```

Comment:
Let's not log an error. In the get phase, just handle the case where the phase is empty, but return Pending